### PR TITLE
chore(metadata-service): add more projects to db (BS-39)

### PR DESCRIPTION
**Pull request by mpro7**
resolves BS-39

First 2 commits are about previous mobile first task - few minor things needed to be updated.
I've went through all projects data we have and added all of them only to the `projects` endpoint. These which I was able to get the JSON only were added to `metadata` endpoint - we don't want to see JSON-LD format here right? ;)

Btw. I think both endpoints we have could be easily combined into one. Do you agree?

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
Nice, thanks. That looks like it was a lot of copy-pasting...

**Comment by BalduinLandolt** [comment]
I agree, the two endpoints _could_ easily be combined. However, I'm not sure they _should_ be combined.  
I kept them separate in order to mimic the API as closely as possible, that we'll eventually have to query (without really knowing how that API will look like). From my understanding, we'll eventually have to get the projects from the `v2/projects` endpoint, but the metadata itself from the newly created `metadata` service.  
That's why I kept them separate. But I'm not sure that makes any sense... What do you think? And maybe @subotic has an opinion on that too?

**Comment by mpro7** [comment]
> I agree, the two endpoints _could_ easily be combined. However, I'm not sure they _should_ be combined.
> I kept them separate in order to mimic the API as closely as possible, that we'll eventually have to query (without really knowing how that API will look like). From my understanding, we'll eventually have to get the projects from the `v2/projects` endpoint, but the metadata itself from the newly created `metadata` service.
> That's why I kept them separate. But I'm not sure that makes any sense... What do you think? And maybe @subotic has an opinion on that too?

I think about merge only in the scope of our fake BE, which is the temporal solution. I think we will not use both endpoints and soon we will have the real BE implementation, so I would like to simplify it.

**Comment by BalduinLandolt** [comment]
> 
> 
> > I agree, the two endpoints _could_ easily be combined. However, I'm not sure they _should_ be combined.
> > I kept them separate in order to mimic the API as closely as possible, that we'll eventually have to query (without really knowing how that API will look like). From my understanding, we'll eventually have to get the projects from the `v2/projects` endpoint, but the metadata itself from the newly created `metadata` service.
> > That's why I kept them separate. But I'm not sure that makes any sense... What do you think? And maybe @subotic has an opinion on that too?
> 
> I think about merge only in the scope of our fake BE, which is the temporal solution. I think we will not use both endpoints and soon we will have the real BE implementation, so I would like to simplify it.

Fine by me. The simpler, the better!

File: services/metadata/backend/data/db.json [hunk 1]
```diff
@@ -1223,6 +1184,527 @@
...
...ren, Publizieren und Archivieren geoarchäologischer Proben. Darüber hinaus sind mikromorphologische Befunde mit den Dünnschliffscans und dadurch mit den Blockproben und Profilen verknüpft. Durch diese Vernetzung wird eine multidimensionale Publikation und Archivierung mikromorphologischer Bilddaten und ihr Bezug zum originalen Feldbefund möglich.\n\nE: The geoarchaeological collection of the IPAS (Integrative Prehistory and Archaeological Science) at the University of Basel includes block and sediment samples of soils and archaeological deposits as well as rocks, flints, ceramics etc. The „I-GEOARCHive“ image database of thin section scans (and corresponding microscope photos) makes the collection digitally accessible to the public and in particular to research and teaching. The freely accessible „I-GEOARCHive“ database enables users to keyword, analyse, publish and archive geoarchaeological samples. Furthermore, micromorphological observations are linked to the thin section scans and thus to the block samples and profiles. This cross-linking enables multidimensional publication and archiving of micromorphological image data and their relation to original archaeological features.",
```
Comment (BalduinLandolt) [inline services/metadata/backend/data/db.json:1191]: this makes me wonder: should we really have `description` here, if that's copy-pasted from the metadata anyways? shouldn't it rather be pulled from the metadata directly?

**Review by BalduinLandolt — COMMENTED** [review]
I'm wondering if the descriptions still make sense in the structure we have now.

File: services/metadata/backend/data/db.json (hunk 1 above)
Comment (mpro7) [reply services/metadata/backend/data/db.json:1191]: Both `name` and `description` are. You're right we should have them from thee, but as for temp solution is easier to have it like that. Imo it doesn't make sense to implement any logic until we will decide on the **final** API and data model shapes (flat/nested, etc.).

Comment (BalduinLandolt) [reply services/metadata/backend/data/db.json:1191]: fair enough.  
in that case, feel free to merge

**Comment by subotic** [comment]
> I agree, the two endpoints _could_ easily be combined. However, I'm not sure they _should_ be combined.
> I kept them separate in order to mimic the API as closely as possible, that we'll eventually have to query (without really knowing how that API will look like). From my understanding, we'll eventually have to get the projects from the `v2/projects` endpoint, but the metadata itself from the newly created `metadata` service.
> That's why I kept them separate. But I'm not sure that makes any sense... What do you think? And maybe @subotic has an opinion on that too?

A bit late to the party, but here is my opinion on this. Define the endpoints in such a way, so that it is easiest to be used by the frontend. It doesn't matter from where what data will come from.

**Pull request merged**